.github/workflows/continuous-integration.yaml

```diff
@@ -63,8 +63,8 @@ jobs:
     needs: [ build ]
     runs-on: ubuntu-latest
 
-    # max run time 20 minutes
-    timeout-minutes: 20
+    # max run time 30 minutes
```

Comment:
Similar to unit tests, integration tests take longer -- should also not have increased a lot but here we are.

Reply:
Not needed, will revert